### PR TITLE
Feat: M2-1784 - Builder - Items Flow (Conditional Logic)

### PR DESCRIPTION
**Pull request by muhurov**
image

### CODE REVIEW

File: src/modules/Builder/features/ActivityItemsFlow/RemoveItemFlowPopup/RemoveItemFlowPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      secondBtnText={t('cancel')}
+      submitBtnColor="error"
+      onSecondBtnSubmit={onClose}
+    >
+      <StyledModalWrapper>
+        <Trans key="removeConditionalPopupDescription">
+          Are you sure you want to remove{' '}
+          <strong>
+            Conditional <>{{ index }}</>
+          </strong>{' '}
```
Comment (alex-i-e) [inline src/modules/Builder/features/ActivityItemsFlow/RemoveItemFlowPopup/RemoveItemFlowPopup.tsx:33]: can you remove redundant space?

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.styles.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { styled } from '@mui/material';
+
+import { StyledFlexTopCenter, variables, theme } from 'shared/styles';
+import { SelectController } from 'shared/components/FormComponents';
+
+export const StyledSummaryRow = styled(StyledFlexTopCenter)`
+  gap: 0.4rem;
+  padding: ${theme.spacing(0.8, 1.2)};
+
+  & > * {
```
Comment (alex-i-e) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.styles.ts:10]: is it possiple to specify with more restricted rules?

File: src/modules/Builder/components/Condition/Condition.styles.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import { StyledFlexTopCenter, variables, theme } from 'shared/styles';
+import { SelectController, InputController } from 'shared/components/FormComponents';
+
+export const StyledCondition = styled(StyledFlexTopCenter)`
+  gap: 0.4rem;
+  padding: ${theme.spacing(0.8, 1.2)};
+  border-radius: ${variables.borderRadius.xs};
+  position: relative;
+
+  & > * {
```
Comment (alex-i-e) [inline src/modules/Builder/components/Condition/Condition.styles.ts:12]: is it possible to use more specific rules?

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Builder/features/ActivityItemsFlow/RemoveItemFlowPopup/RemoveItemFlowPopup.tsx (hunk 1 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/RemoveItemFlowPopup/RemoveItemFlowPopup.tsx]: they are not a redundant, they can be added manually

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.styles.ts (hunk 2 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.styles.ts]: done

File: src/modules/Builder/components/Condition/Condition.styles.ts (hunk 3 above)
Comment (muhurov) [reply src/modules/Builder/components/Condition/Condition.styles.ts]: done

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ConditionRow/ConditionRow.utils.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    case ItemResponseType.MultipleSelection:
+      return ConditionItemType.MultiSelection;
+    default:
+      return ConditionItemType.SingleSelection;
+  }
+};
+
+export const getItemOptions = (items: ItemFormValues[]) =>
+  items?.reduce(
+    (optionList: { labelKey: string; value: string; type: ConditionItemType }[], item) => {
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ConditionRow/ConditionRow.utils.ts:33]: pls move type of optionList to ConditionRow.types.ts

Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ConditionRow/ConditionRow.utils.ts]: done

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**